src/views/blocks/attendees/view-link.php

```diff
@@ -34,16 +35,24 @@
 $is_event_page = class_exists( 'Tribe__Events__Main' ) && Tribe__Events__Main::POSTTYPE === $event->post_type;
 
 $events_label_singular = $post_type->labels->singular_name;
-$counters              = array();
+$counters              = [];
 $rsvp_count            = $view->count_rsvp_attendees( $event_id, $user_id );
 $ticket_count          = $view->count_ticket_attendees( $event_id, $user_id );
 
-if ( 0 !== $rsvp_count ) {
-	$counters[] = sprintf( _n( '%d %s', '%d %s', $rsvp_count, 'event-tickets' ), $rsvp_count, _nx( 'RSVP', 'RSVPs', $rsvp_count, 'Singular and plural texts for RSVP(s)', 'event-tickets' ) );
+if ( 1 === $rsvp_count ) {
+	$counters[] = sprintf( _x( '%d %s', 'RSVP count singular', 'event-tickets' ), $rsvp_count, tribe_get_rsvp_label_singular( basename( __FILE__ ) ) );
+}
+
+if ( 1 < $rsvp_count ) {
```

Comment:
```suggestion
elseif ( 1 < $rsvp_count ) {
```

Can you bump this up to directly after the `1 === $rsvp_count` logic conditional block?